docs/using_python_client_with_hazelcast.rst

```diff
@@ -378,6 +378,76 @@ A Reliable Topic usage example is shown below.
     # Publish a message to the Topic
     topic.publish("Hello to distributed world")
 
+Using ReliableMessageListener
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`ReliableMessageListener
+<https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/topic/ReliableMessageListener.java>`__ is a `MessageListener
+<https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/topic/MessageListener.java>`__ to better integrate with the reliable topic.
+
+If a regular MessageListener is registered on a reliable topic, the message listener works fine, but it can't do much more than listen to messages.
+If a ReliableMessageListener is registered on a normal topic, only the MessageListener methods are called.
+
+The following is an example Reliable Message Listener class.
+
+.. code:: python
+
+    class MyListener(ReliableMessageListener):
+        def on_message(self, message):
+            print("Received new message: ", message)
+
+        def retrieve_initial_sequence(self):
+            print("Listener function retrieve_initial_sequence is called")
+            return 0
+
+        def store_sequence(self, sequence):
+            print("Listener function store_sequence is called with sequence: ", sequence)
+            pass
+
+        def is_loss_tolerant(self):
+            print("Listener function is_loss_tolerant is called")
+            return True
+
+        def is_terminal(self, error):
+            print("Listener function is_terminal is called with error: ", error)
+            return False
+
+        def on_cancel(self):
+            print("Listener function on_cancel is called")
+
+Durable Subscription
+********************
+
+The ReliableMessageListener allows you to control where you want to start processing a message when the listener is registered. This makes it possible to create a durable subscription by storing the sequence of the last message and using this sequenceId as the sequenceId to start from.
+
+Exception Handling
+******************
+
+The ReliableMessageListener also gives the ability to deal with exceptions using the `is_terminal(error)` method. This method allows you to control which exceptions should terminate the execution of the listener and cancel it. If a plain MessageListener is used, it won't terminate on exceptions and it will keep on running. But in some cases it is better to stop running.
+
+Global Order
+************
+
+The ReliableMessageListener will always get all events in order (global order). It will not get duplicates and there will only be gaps (loss of messages) if it is too slow. For more information see `is_loss_tolerant()`.
```

Comment:
Is it possible to give a link to the methods you refer? like is_loss_tolerant here